assets/js/modules/analytics-4/datastore/webdatastreams.test.js

```diff
@@ -71,6 +72,17 @@ describe( 'modules/analytics-4 webdatastreams', () => {
 		displayName: 'Another datastream',
 	};
 
+	const webDataStreamDotCom2 = {
+		name: 'properties/1000/dataStreams/2002',
+		webStreamData: {
+			measurementId: '1A2BCD347E', // eslint-disable-line sitekit/acronym-case
```

Comment:
This isn't a valid measurement ID, is it? I realize this follows the others above, but these should probably use more realistic valid values by adding the `G-` prefix.

Reply:
If I remember correctly, measurement IDs without `G-` prefix was used at the beginning when GA4 API had been published. I think, if we want to update all measurementIDs to have the `G-` prefix, then it is worth a separate ticket. What do you think?

Reply:
I remember something about that as well, but our own validation utility does check for the `G-` prefix 🤔 https://github.com/google/site-kit-wp/blob/5fc5b8e85ad32e4616038b527307d2145dabe4fc/assets/js/modules/analytics-4/utils/validation.js#L91-L96